core/deepmind.py

```
@@ -427,6 +427,34 @@ async def organize_and_inject_memories(
+                            try:
+                                ledger_platform = str(event.get_platform_name() or "").strip()
+                            except Exception:
+                                ledger_platform = ""
+                        memory_sql_manager.upsert_user(
+                            ledger_platform, ledger_sender_id, ledger_sender_name
+                        )
+                        ledger_group_id = ""
+                        if hasattr(event, "get_group_id"):
+                            try:
+                                ledger_group_id = str(event.get_group_id() or "").strip()
+                            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)